index.html

```diff
@@ -0,0 +1,130 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/fontawesome.min.css" integrity="sha512-SgaqKKxJDQ/tAUAAXzvxZz33rmn7leYDYfBP+YoMRSENhf3zJyx3SBASt/OfeQwBHA1nxMis7mM3EV/oYT6Fdw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="index.css">
+    <title>HennaHub</title>
+</head>
+<body>
+    <header>
+     <i class="fa-solid fa-bars" aria-hidden="true"></i>
+        <div class="top-header">
+            <ul>
+                <li><a href="https://web.facebook.com/hennahubuk/?_rdc=1&_rdr"><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <div class="sub-header">
+            <a href="https://hennahubuk.business.site/"><img src="./images/logo.jfif" alt="website_logo"></a>
+      <div class="header-right">
+        <nav>
+          <ul class="links">
+            <li><a class="nav-link active" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#">Program</a></li>
+            <li><a class="nav-link" href="#">Join</a></li>
+            <li><a class="nav-link" href="#">Sponsor</a></li>
+            <i class="fa-solid fa-xmark"></i>
+          </ul>
+          <Span>Special Seat</Span>
+        </nav>
+        </div>
+    </header>
+    <div class="begin">
+        <div class="introduction">
+            <div class="intro-heading">
+                <p>"Hello HennaHub Globe!"</p>
+                <h1 class="intro-trans-anim">HENNAHUBUK'S CREATIVE DESIGN APPRENTICE SUMMIT <br>2023</h1>
+            </div>
+            <p class="intro-text">A joyful celebration of creativity and mehndi design for the first apprentice graduation og HennaHubUK academy</p>
+            <h2>2023-12-20(WED) ~ 21(THU)</h2>
+            <span>@ 1312 Gallowgate Glasgow G31 4DR UK</span>
+        </div>
+    </div>
+    <div class="main-program">
+        <div>
+        <h2>Main Program</h2>
+        <hr>
+    </div>
+        <div class="program">
+            <section class="program-list">
+                <div class="card-display">
+                <div class="program-image">
+                    <img src="./images/exhibitionicon (2).svg" alt="lecture icon">
+                    <p class="program-name">Lecture</p>
+                    <p class="program-text">
+                        Listen to honourable speakers from various locations about the mission of hennahubuh mehndi creations 
+                    </p> 
+                </div>
```

Comment:
- [ ] Kindly wrap each program in an `article` tag for semantics reasons. You can read more about the `article` tag [here](https://www.w3schools.com/tags/tag_article.asp)